### PR TITLE
Feat: Refactor parameter names For Products

### PR DESCRIPTION
**Pull request by wangerekaharun**
… biometric KYC

Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ class SmileIDDocumentVerificationEnhancedView(context: Context, appContext: AppC
     private val onResult by EventDispatcher()
     private val onError by EventDispatcher()
 
-    fun updateConfig(config: EnhancedDocumentVerificationRequest) {
+    fun updateConfig(config: EnhancedDocumentVerificationRecord) {
```
Comment (jumaallan) [inline android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt:33]: I think `Request` suffix makes more sense than using `Record`

**Review by jumaallan — COMMENTED** [review]
Left some comments - will review and test tomorrow

File: android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt (hunk 1 above)
Comment (wangerekaharun) [reply android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt:33]: I don't think so in this case - cause they're params and we're not making any request per se

Comment (jumaallan) [reply android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt:33]: maybe call them `Params` then? since they are params

Comment (wangerekaharun) [reply android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt:33]: Sure let me rename

File: README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ initialize(true);
 
 ```typescript
 // Create configuration object with your Smile ID portal credentials
-const config = new ExpoConfig(
+const config = new SmileConfig(
```
[the remark this answers is not in the document]
Comment (wangerekaharun) [reply README.md:96]: This isn't clear

File: sample-expo/HomeScreen.tsx [hunk 3]
```diff
@@ -68,37 +68,107 @@ const PRODUCTS = [
   }
 ];
 
-const config = new ExpoConfig(
+const smileConfig = new SmileConfig(
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply sample-expo/HomeScreen.tsx:71]: I meant to say - name it config like you did on the comment where you say its not clear
